lib/s3-s3.js

```diff
@@ -83,8 +107,9 @@ var s3 = function (primaryS3, secondaryS3) {
      */
     getObject: function() {
       noArguments(arguments, 'getObject');
```

Comment:
Is there anyway to have an option specifically check the secondary instead of the primary?  Would be very useful in the work I'm doing right now!